app/assets/stylesheets/admin/forms/forms.scss

```diff
@@ -44,3 +44,124 @@ textarea {
     float: left;
   }
 }
+
+.date.required.form-control {
+  display: inline-block;
+  width: 32%;
+  margin-right: 5px;
+}
+
+//
+//project step form
+//
```

Comment:
The CSS below here should probably go in a different file. I usually consider commented headers in a CSS file like this to be an anti-pattern.